domains/api/http/transport.go

```diff
@@ -84,7 +86,59 @@ func MakeHandler(svc domains.Service, authn authn.Authentication, mux *chi.Mux,
 		})
 	})
 
-	mux.Get("/health", supermq.Health("auth", instanceID))
+	mux.Route("/{domainID}/invitations", func(r chi.Router) {
+		r.Use(api.AuthenticateMiddleware(authn, true))
+		r.Post("/", otelhttp.NewHandler(kithttp.NewServer(
+			sendInvitationEndpoint(svc),
+			decodeSendInvitationReq,
+			api.EncodeResponse,
+			opts...,
+		), "send_invitation").ServeHTTP)
+		r.Get("/", otelhttp.NewHandler(kithttp.NewServer(
+			listDomainInvitationsEndpoint(svc),
+			decodeListInvitationsReq,
+			api.EncodeResponse,
+			opts...,
+		), "list_invitations").ServeHTTP)
+		r.Route("/{userID}", func(r chi.Router) {
+			r.Get("/", otelhttp.NewHandler(kithttp.NewServer(
+				viewInvitationEndpoint(svc),
+				decodeInvitationReq,
+				api.EncodeResponse,
+				opts...,
+			), "view_invitation").ServeHTTP)
+			r.Delete("/", otelhttp.NewHandler(kithttp.NewServer(
+				deleteInvitationEndpoint(svc),
+				decodeInvitationReq,
+				api.EncodeResponse,
+				opts...,
+			), "delete_invitation").ServeHTTP)
+		})
+	})
```

Comment:
@felixgateru Can we add `domains` prefix ?  Because invitations in domains service, it is part of particular domain like domain roles